src/Ushahidi/Modules/V5/Commands/Tos/CreateTosCommand.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Ushahidi\Modules\V5\Commands\Tos;
+
+use Ushahidi\Modules\V5\Commands\CreateCommand;
+use Ushahidi\Modules\V5\Models\Tos;
+
+class CreateTosCommand extends CreateCommand
```

Comment:
See - because of 'generic' command you don't know what type of data you're working with here. Consider Commands and Queries to be DTO's that hold input data and input data only.